lldb/test/API/lang/swift/async/unwind/unwind_register_pressure/main.swift

```diff
@@ -0,0 +1,65 @@
+func work(_ objects: Any...) {
+  for object in objects {
+    print("Processing object of type: \(type(of: object))")
+  }
+}
+
+func use(_ x: Int, _ y: Int) -> Int {
+    return x &* y &+ x &- y
+}
+
+var arr: [Int] = []
+
+func ASYNC___1___() async -> Int {
+  var a1 = 1, a2 = 2, a3 = 3, a4 = 4, a5 = 5 // BREAK HERE
```

Comment:
```suggestion
  var a1 = 1, a2 = 2, a3 = 3, a4 = 4, a5 = 5 
  print("BREAK HERE")
```
Otherwise it's kind of ambiguous where exactly the breakpoint will be because the above line can result in vastly different code depending on the target.